pulpcore/cli/container/context.py

```diff
@@ -110,6 +110,39 @@ class PulpContainerRepositoryContext(PulpContainerBaseRepositoryContext):
         "roles": [PluginRequirement("container", "2.11.0.dev")],
     }
 
+    def modify(
+        self,
+        href: str,
+        add_content: Optional[List[str]] = None,
+        remove_content: Optional[List[str]] = None,
+        base_version: Optional[str] = None,
+    ) -> Any:
+        if remove_content:
+            self.call(
+                "remove", parameters={self.HREF: href}, body={"content_units": remove_content}
+            )
+        if add_content:
+            self.call("add", parameters={self.HREF: href}, body={"content_units": add_content})
```

Comment:
To which class, the `PulpContainerBaseRepositoryContext`? I don't think so since only the regular container repository can call the add and remove endpoints. Also this modify call is needed since that is what the content factory uses to create the add and remove commands.

Reply:
I was really thinking about `PulpRepositoryContext` now that you say that push-repos cannot modify content, this would add the need for a capability. Maybe not worth the trouble now.